code/LINDA/LINDA_fire.dm

```diff
@@ -152,18 +152,18 @@
 // Garbage collect itself by nulling reference to it
 
 /obj/effect/hotspot/proc/Kill()
-	air_master.hotspots -= src
-	DestroyTurf()
-	qdel(src)
+	PlaceInPool(src)
```

Comment:
Yes, I updated my comment; Those should have been in Destroy() in the first place, and letting Kill() call PlaceInPool directly is fine, albiet it doesn't matter- Now that everything is in Destroy(), qdel or PlaceInPool will both work just fine, but in general everything should attempt to qdel() hotspots, not call an arbitrary variable or manually pool them